data/json/vehicles/military.json

```diff
@@ -714,5 +714,16 @@
       { "x": 0, "y": 2, "chance": 2, "item_groups": [ "mil_accessories" ] },
       { "x": 0, "y": 2, "chance": 2, "item_groups": [ "gunmod_milspec" ] }
     ]
+  },
+  {
+    "id": "cannon_3in",
+    "type": "vehicle",
+    "name": "cannon",
+    "blueprint": [ "HUH" ],
+    "parts": [
+      { "x": 0, "y": 0, "parts": [ "frame_wood", "turret_mount", "mounted_3in_ordnance_rifle" ] },
```

Comment:
You did not define `mounted_3in_ordnance_rifle` vehicle part.

Reply:
Hmm, must've missed that moving from my testing environment